server/upstream/upstream-tarballs.txt

```diff
@@ -355,6 +355,7 @@ ibus-unikey:google:ibus-unikey
 iceauth:httpls:http://xorg.freedesktop.org/releases/individual/app/
 ico:httpls:http://xorg.freedesktop.org/releases/individual/app/
 icon-naming-utils:httpls:http://tango.freedesktop.org/releases/
+iio-sensor-proxy:httpls:https://github.com/hadess/iio-sensor-proxy/releases
```

Comment:
interesting - there were already one link like that in the file, that's why I did not see it as a problem.

e.g.:
osm-gps-map:httpls:https://github.com/nzjrs/osm-gps-map/releases/